### ISSUE TITLE
UFFTYPER: Unrecognized atom type: Ca+2 (0)

### ISSUE THREAD

**Issue by Ruibin-Liu** [chaidiscovery/chai-lab#194]
I am trying to put ions to fold together with protein chains. Here is how I did it:
```
>ligand|name=ca_1
[Ca+2]
```
but I got the error `UFFTYPER: Unrecognized atom type: Ca+2 (0)`. For Mg2+, the error is different but just
`UFFTYPER: Unrecognized charge state for atom: 0`

What is the corrent way?

**Comment by wukevin** [comment]
I believe this error shouldn't affect the correct generation and placement of the Ca ion in the predicted structure. Can you confirm?

**Comment by Ruibin-Liu** [comment]
Yes, I have more than 1 Ca2+ ions and they are in the correct places for my system. 
A side question, did you test the MMFF14s force field instead of UFF in generating small molecule 3D conformers? Do you think it matters to use a different FF?

**Comment by wukevin** [comment]
I wouldn't expect that using a difference force field for 3D conformers would make a large difference in predictions, though we have not tested this empirically.

We've also added a bit of testing around the error you observed, making sure that the actual charge is and atomic element are passed through correctly.

**Comment by Ruibin-Liu** [comment]
Thanks!
